backend/debug/dwarf/dwarf_flags/dwarf_flags.ml

```diff
@@ -136,3 +136,5 @@ let gdwarf_format = ref default_gdwarf_format
 let default_gdwarf_self_tail_calls = true
 
 let gdwarf_self_tail_calls = ref default_gdwarf_self_tail_calls
+
+let gdwarf_may_alter_codegen = ref true
```

Comment:
(Just want to be sure it is enabled by
default on purpose.)

Reply:
Actually not, this should be false!